app/pages/projects.cjsx

```diff
@@ -38,7 +35,7 @@ ProjectsPage = React.createClass
         delete query[key]
     newLocation = Object.assign {}, @props.location, {query}
     newLocation.search = ""
-    @props.history.replace newLocation
```

Comment:
The only other thing on this - we're replacing `replace` with `push`, but I prefer it that way anyway.